test/test_quantization.py

```diff
@@ -2,10 +2,14 @@
 
 from torch.testing._internal.common_utils import run_tests
 
-# Quantized Tensor
-from quantization.core.test_quantized_tensor import TestQuantizedTensor  # noqa: F401
-# Quantized Op
-# TODO: merge test cases in quantization.test_quantized
+# Quantization core tests. These include tests for
+# - quantized kernels
+# - quantized functional operators
+# - quantized workflow modules
+# - quantized workflow operators
+# - quantized tensor
+
+# 1. Quantized Kernels
 from quantization.core.test_quantized_op import TestQuantizedOps  # noqa: F401
 from quantization.core.test_quantized_op import TestQNNPackOps  # noqa: F401
 from quantization.core.test_quantized_op import TestQuantizedLinear  # noqa: F401
```

Comment:
I feel ideally we should merge these into TestQuantizedOps actually.

Reply:
Yeah I think we can have a unified class for all ops. I'll add a todo